### DISCUSSION TITLE
Modules and Understanding (Reticulum)

### DISCUSSION THREAD

**Discussion by jacqueszsx** [markqvist/Reticulum#255] [General]
Hi to everyone
I am new to the group busy studying the documentation hope I am on the right track

Keen to set the following up
- Unified integrated messaging (live, store & forward, post-box "unattended") over HF, UHF/VHF/Lora and I2P
- Be able to use weak signals (HF)
- Primarily for Messaging ex Sideband

Can the following newer hardware be used
- Can a SX1262 Lora HAT for Raspberry Pi be used in place of LilyGO LoRa32 v2.1 board

Attached my high level understanding so, hope it is correct? Please comment for example not sure if all to be installed on diagram

[Reticulum.pdf](https://github.com/markqvist/Reticulum/files/10948113/Reticulum.pdf)

, Hope you will be able to assist Regards J

**Comment by pizzadoglover911** [comment]
> Can a SX1262 Lora HAT for Raspberry Pi be used in place of LilyGO LoRa32 v2.1 board

I can comment on this portion, from what I can tell, SX1262 chip is not yet supported. It appears future support for other chips is planned/in progress. This page could be useful if you have not seen it yet: https://unsigned.io/guides/2022_01_26_how-to-make-your-own-rnodes.html. Any supported chip would work fine(I chose SX1276 on a Heltec board, for RNode) .


Your plan seems fair. That raspberry pi will put in a workout!

**Reply by jacqueszsx** [reply]
Thank you, will read the RNode setup url. Good point re raspberry pi, idea is to plug and play the communication transportation layer/s devices and not use all at the same time with a pi. Would be great if we could develop as part of Reticulum utility the config of ex QDX Tranceiver, OpenModem, Digirig ..... other

**Reply by markqvist** [reply]
I am working on SX1262 support. Community has kindly donated the necessary hardware for implementing support for mostly all boards based on the SX1262 chip. Currently it's mostly an issue of being able to dedicate sufficient time to it, since it is not a trivial task.

No good open-source drivers exist for the SX1262 chips, that provide the low-level access to the hardware that the RNode firmware needs, so realistically, we are left with writing a new SX1262 driver from scratch. This approach will give the best result anyways, but a significant amount of work and time needs to be put into it.
